### ISSUE TITLE
RAT proof feature is showing test results as invalid only after 48h and not after 24h 

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-documentation#768]
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug affects both Android **and** iOS, for specific issues / questions that apply only to one operating system, please raise them in the respective repositories:
    * [Android repository](https://github.com/corona-warn-app/cwa-app-android)
    * [iOS repository](https://github.com/corona-warn-app/cwa-app-ios)
* [X] Bug is not already reported in another issue

## Technical details

- Device name: Not important
- iOS/Android version: Not important
- App version: Any since 2.1

## Describe the bug

The RAT proof feature in the app, for rapid antigen test results (not DCCs!), is showing a test result as invalid after 48. However, in Germany, according to https://www.zusammengegencorona.de/testen/schnelltest/ a RAT is only valid for 24h.

## Steps to reproduce the issue

1. Add a negativ RAT to the app
2. Wait for 48h to pass
3. See that the app only after 48h shows the test result as invalid and not after 24h

## Expected behaviour

The app should already shown a test result as invalid after 24h. 

---
Internal Tracking ID: [EXPOSUREAPP-10853](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10853)

**Comment by mlenkeit** [comment]
@Ein-Tim thanks for raising this. 

This currently works as intended. When the feature was designed, RAT did have a validity of up to 48 hours (see e.g. [1]) depending on local regulations and exceptions. We would need to check if this has been adjusted nation wide or if there are other reasons to keep it at 48 hours.

1: https://www.mdr.de/nachrichten/thueringen/ost-thueringen/saale-orla/apotheke-schleiz-corona-schnelltest-100.html

**Comment by mlenkeit** [comment]
One reason to keep the 48 hours might be the fact that the rules for entering Germany say 48 hours:

> Alternativ ein negatives Ergebnis eines Antigen-Schnelltests in deutscher, englischer, französischer, italienischer oder spanischer Sprache. Der Test bei einem Voraufenthalt in einem Hochrisikogebiet darf höchstens 48 Stunden vor der Einreise erfolgen. 

Quelle: https://www.auswaertiges-amt.de/de/quarantaene-einreise/2371468

**Comment by Ein-Tim** [comment]
@mlenkeit

Thanks for the feedback, I see what you mean! However, please also understand the following:
I opened this issue after my mother wanted to visit a public swimming pool, where currently 2G+ is in place. She did not inform herself whether a RAT is allowed to be, because she knew that CWA would show a too old test result as invalid. The test was not yet older than 48h, so my mother thought that she would be good to go.
Then there, at the public swimming pool, the personell which checked the 2G+ certificates, said that a RAT is not allowed to be older than 24h.
My mother was very annoyed and it took me a good amount of time to convince her not to uninstall the CWA 😅

So, long speech short, something has to be done, either by improving the text shown drastically, removing the feature that RATs are getting invalid after 48h or making them expire after 24h.

Thanks again for taking care of this!

**Comment by Ein-Tim** [comment]
Additionally the fact that one can enter Germany with a RAT older than 48h is not really a good argument as one normally does not get a QR code for adding a test result (**not** an EU DCC) to the CWA.

**Comment by Ein-Tim** [comment]
@mlenkeit 

As CWA seems to follow the DCC business rules for Germany (see https://github.com/corona-warn-app/cwa-app-ios/pull/3924#issuecomment-984772269), do you know who is in charge for updating them? 
The team over https://github.com/Digitaler-Impfnachweis or the CWA team?

**Comment by Ein-Tim** [comment]
This issue becomes worse in version 2.16.

The [blog post](https://www.coronawarn.app/de/blog/2022-01-17-cwa-2-16/) states (and this is also implemented in the app):
>	3G – Steht für vollständig geimpft, genesen oder negativ getestet. Nach der 3G-Regel müssen Nutzer*innen eins der folgenden Zertifikate nachweisen: Impf- oder Genesenenzertifikat, Schnelltestzertifikat (maximal 48 Stunden alt) oder PCR-Testzertifikat (maximal 72 Stunden alt).

While https://www.zusammengegencorona.de/informieren/alltag-und-reisen/aktuelle-regelungen/ clearly states:
> Die sogenannte 3G-Regel steht für „geimpft, genesen oder getestet“. Wer nicht vollständig geimpft ist oder nicht als genesen gilt (mehr dazu hier) muss in bestimmen Fällen entweder einen negativen Schnelltest (maximal 24 Stunden alt) oder einen negativen PCR-Test (maximal 48 Stunden alt) vorlegen.

@mlenkeit is there any change planned here? This is IHMO really a top priority issue.

**Comment by Ein-Tim** [comment]
Just to point out the consequences of this issue: 
People are thinking, they have e.g. the 2G+ status, as CWA shows this, however, they are then rejected entry as CWA displays **wrong** information.

I could understand every user uninstalling CWA after having made this experience.

**Comment by NicoErnst** [comment]
I second Tims observation. This issue is being brought up more and more by users on Twitter on various ways and all we can say ist: We know, it´s committed, dunno when this will be fixed :(

**Comment by Ein-Tim** [comment]
Closing in favor of https://github.com/corona-warn-app/cwa-documentation/issues/819.
